v2/tools/generator/internal/interfaces/kubernetes_resource_interface.go

```diff
@@ -400,7 +400,7 @@ func createResourceReference(
 			"ResourceReference"))
 	compositeLitDetails.AddField("Group", group)
 	compositeLitDetails.AddField("Kind", kind)
-	compositeLitDetails.AddField("Namespace", astbuilder.Selector(dst.NewIdent(receiverIdent), "Namespace"))
+	// compositeLitDetails.AddField("Namespace", astbuilder.Selector(dst.NewIdent(receiverIdent), "Namespace"))
```

Comment:
This can go now - presumably left from initial steps.

Reply:
Oh, yep - good catch